spec/requests/post_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  before do
+    @user = User.create(name: 'Binyam', photo: 'url', bio: 'I am a software developer.')
+    @post = Post.create(author: @user, title: 'Hello', text: 'Good career choice.', comments_count: 3, likes_count: 4)
+  end
+
+  describe 'GET /index' do
+    it 'returns http success for posts#index action' do
+      get user_posts_url(user_id: @user.id)
+      expect(response).to have_http_status(:success) # 200
+      expect(response.body).to include('Here is a list of posts for a given user')
+    end
```

Comment:
- [ ] Please make sure to add the below test for the user index action as requested in the project requirements.

![image](https://github.com/binyamolango/Blog-App2/assets/83803414/cfe2fac3-e1e5-47b5-9215-862518548804)